app/controllers/rooms_controller.rb

```diff
@@ -74,10 +74,12 @@ def increment_score
       return
     end
 
-
     team = params[:team].downcase
 
-    raise("Double Score") if @room.increment_at && @room.increment_at > 1.seconds.ago
+    # Times when this call shouldn't happen
+    if @room.increment_at && @room.increment_at > 1.seconds.ago
+      return
+    end
```

Comment:
heh, you could have replaced raise() with return (to keep it a one liner). you don't have to change it though.